src/schemas/json/project.json

```diff
@@ -85,8 +85,8 @@
 				"uniqueItems": true
 			}
 		},
-		"bundleExclude": {
-			"description": "List of files to exclude from publish output (kpm bundle).",
+		"publishExclude": {
+			"description": "List of files to exclude from publish output (dnu publish).",
```

Comment:
`List of files/folders to exclude from ...`.
You can specify folders here and all contents of the folders get excluded. We have this usage in our template project:
https://github.com/aspnet/Templates/blob/release/src/BaseTemplates/StarterWeb/project.json#L34